src/test/java/uk/ac/cam/cl/dtg/isaac/api/managers/EventBookingManagerTest.java

```diff
@@ -93,7 +93,7 @@ public void requestBooking_checkTeacherAllowedOnStudentEventDespiteCapacityFull_
         dummyEventBookingPersistenceManager.acquireDistributedLock(testEvent.getId());
         expectLastCall().atLeastOnce();
 
-        expect(dummyEventBookingPersistenceManager.createBooking(testEvent.getId(), someUser.getId(), BookingStatus
+        expect(dummyEventBookingPersistenceManager.createBooking(testEvent.getId(), someUser.getId(), null, BookingStatus
```

Comment:
Would be good to have a couple tests to make sure reserved bookings behave properly when events are full / nearly full.